internal/stdlib/execenv/execenv_windows.go

```diff
@@ -25,14 +25,14 @@ import (
 // will be sourced from syscall.Environ().
 func Default(sys *syscall.SysProcAttr) (env []string, err error) {
 	if sys == nil || sys.Token == 0 {
-		return syscall.Environ(), nil
+		return windows.Environ(), nil
```

Comment:
Does this not build without replacing `syscall` with `windows`? Just wondering if we can avoid that so as to keep deviation from upstream as little as possible.

Reply:
No this would build fine, I swapped to Windows as there's a x/sys/Windows replacement/mirror for everything in syscall and technically the syscall package is locked and some of it is deprecated. I can swap to syscall if preferred. @anmaxvl What do you think?

Reply:
I misread the original code a bit, I thought it was `internal/syscall` and we replaced it with `windows`, turns out it was originally `internal/syscall/windows` and `syscall` was imported on its  own. 
I agree with @ambarve that we should probably keep `syscall` here.

Reply:
Will try and add everything thats touched in and update, I'll ping when done!